.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Custom issue template
```

Comment:
Is there a config in the settings that list the issue templates, or will any '.md' file in "ISSUE_TEMPLATE" folder be picked up?